ras/ras.py

```diff
@@ -146,7 +146,7 @@ def test6_lsslot(self):
                       "==")
         self.run_cmd("lsslot")
         self.run_cmd("lsslot -c mem")
-        self.run_cmd("lsslot -ac pci")
+        self.run_cmd("lsslot -c pci")
```

Comment:
@shirishaganta Why is this change needed ? Is the option removed or is it failing ?

Reply:
It is failing on the lpars, which don't have pci slots available

Reply:
```
       -c <connector type>
              Display the slots of the specified connector type.  The valid connector types are "pci" for hotplug PCI slots, "slot" for logical  slots,  "phb"  for  PHB's,
              "port" for LHEA ports, "mem" for memory, and "cpu" for cpu's. The default is "slot" if no -c option is specified.

       -a     Display available slots, valid for "pci" slots only.
```
From the man page, looks like `-a` should not be of much of a difference.

But the main thing is:  What is the reason this is removed from the test ? Because, it is a functionality we are checking, I suggest instead of removing it, handle the failure in cases where it is not applicable and fail / cancel / warn accordingly ?

Reply:
okay will do that